src/modules/organization/components/InternalGroupDetail/components/EditHighlights/InternalGroupUserHighlightEditForm.tsx

```diff
@@ -0,0 +1,117 @@
+import { InternalGroupUserHighlightNode } from 'modules/organization/types'
+import { useEditHighlightAPI } from './useEditHighlightAPI'
+import { useEditHighlightLogic } from './useEditHighlightLogic'
+import { useState } from 'react'
+import {
+  Button,
+  Container,
+  FileInput,
+  Group,
+  Image,
+  SimpleGrid,
+  Stack,
+  Switch,
+  Textarea,
+  TextInput,
+} from '@mantine/core'
+import { InternalGroupSelect, UserSelect } from 'components/Select'
+import { IconPhoto } from '@tabler/icons'
+
+interface InternalGroupUserHighlightEditProps {
+  highlight: InternalGroupUserHighlightNode
+  onCompletedCallback?: () => void
+}
+
+export const InternalGroupUserHighlightEditForm: React.FC<
+  InternalGroupUserHighlightEditProps
+> = ({ highlight, onCompletedCallback }) => {
+  const { form, onSubmit } = useEditHighlightLogic({
+    ...useEditHighlightAPI({ highlight, onCompletedCallback }),
+  })
```

Comment:
`useAPI` already returns an objects, so don't need to spread it inside an object

```suggestion
  const { form, onSubmit } = useEditHighlightLogic(
    useEditHighlightAPI({ highlight, onCompletedCallback }),
  )
```

Reply:
fiexd